components/Storyblok/SbTypeform.tsx

```diff
@@ -0,0 +1,162 @@
+import { storyblokEditable } from '@storyblok/react/rsc';
+import {
+ PopOver, PopUp, SideTab, Slider, Widget,
+} from '@/components/TypeForm';
+import { Heading } from '@/components/Typography';
+import * as CTAStyles from '@/components/Cta/Cta.styles';
+import { cnb } from 'cnbuilder';
+
+export type SbTypeformProps = {
+  blok: {
+    _uid: string;
+    id: string;
+    autoFocus: false;
+    component: string;
+    embedType: 'slider' | 'popup' | 'popover' | 'sidetab' | 'widget';
+    autoResize: boolean;
+    fullScreen: boolean;
+    heading: string;
+    hideFooter: boolean;
+    hideHeader: boolean;
+    buttonLabel: string;
+    embedHeight: string;
+    disableScroll: boolean;
+    enableSandbox: boolean;
+    transitiveSearchParams: string;
+    displayAsFullScreenModal: boolean;
+  };
+};
+
+// TODO: Move this to the non-storyblok Typeform component dir.
+export const styles = {
+  container: 'cc flex flex-col place-items-center rs-py-6',
+  heading: 'px-[40px] tf-sm:px-[80px] max-w-[878px]',
+  fauxCTA: cnb(CTAStyles.cta, CTAStyles.ctaVariants.solid, CTAStyles.ctaSizes.large),
+};
+
+export const SbTypeform = ({
+  blok,
+}: SbTypeformProps) => {
+
+  const {
+    id,
+    autoFocus,
+    embedType,
+    autoResize,
+    fullScreen,
+    heading,
+    hideFooter,
+    hideHeader,
+    buttonLabel,
+    embedHeight,
+    disableScroll,
+    enableSandbox,
+    transitiveSearchParams,
+    displayAsFullScreenModal,
+  } = blok;
+
+  const transitiveParams = transitiveSearchParams.split(`\n`);
+  const height = parseInt(embedHeight, 10);
+  const resize = autoResize || !height;
+
+  switch (embedType) {
+    case 'slider': {
+      return (
+        <div {...storyblokEditable(blok)} className={styles.container}>
+          <Heading as='h2' size={3} className={styles.heading}>{heading}</Heading>
+          <Slider
+            id={id}
+            autoClose={1000}
+            disableScroll={disableScroll}
+            enableSandbox={enableSandbox}
+            hideFooter={hideFooter}
+            hideHeaders={hideHeader}
+            transitiveSearchParams={transitiveParams}
+            displayAsFullScreenModal={displayAsFullScreenModal}
+          >
+            <span className={styles.fauxCTA}>{buttonLabel}</span>
```

Comment:
I faked the CTA styles for this embed. Feel free to swap it out with something else.